### ISSUE TITLE
Heredoc is indented on each run

### ISSUE THREAD

**Issue by doits** [prettier/plugin-php#1624]
Following happens when `phpVersion` is set to something `>= 7.3`:

The heredoc gets indented on each prettier run:

```php
// input

xxx(
    <<<EOF
    something
    EOF
    ,
);

// run prettier

xxx(
  <<<EOF
      something
      EOF
  ,
);

// run prettier again

xxx(
  <<<EOF
          something
          EOF
  ,
);

// run prettier again

xxx(
  <<<EOF
              something
              EOF
  ,
);
```

It seems to add `tabWidth` spaces to the intendation on each run. Does not happen with `phpVersion <= 7.2`.

[Playground link](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AjAAwMA0IMAhgEblW14AsrAK7YoAFS648AM3YAbUa2zEA5nKgBFIehhR8shVFZhqYAGpJlWfCADsAOn4hWMCO3hzVAYXQBbX+wACgASgfiuQkYgnG4AxlAAyjAAnuo2YNgQALQATM4g0ACOQvDQgW4qAfryiiDEohAw5eyV7NWGAL6sUBTwMOgoaPBYUqDs2DDtol11mGBCk3ggADwA-CZgADqY2wAe+wAUy8cAosgAYtsABDdX2H6w1KrXt2fnVwCUANzbe4cvN2OyzeALuDxoz0wtyuIKht2Y22+v0w+12B1BQNh0Oh918j0h2NeF1BCMw3xAHSAA)

**Comment by czosel** [comment]
Thanks for the bug report @doits!

**Comment by Epskampie** [comment]
This looks like the same bug that was present for strings earlier too: #1593 

I fixed it in #1609 so @doits  you might want to take a look at that, and make a similar PR to fix this.
